content/creator/sdk7/3d-essentials/entity-positioning.md

```diff
@@ -395,15 +395,43 @@ When creating an `AvatarAttach` component, pass an object with the following dat
 - `avatarId`: _Optional_ The ID of the player to attach to. This is the same as the player's Ethereum address, for those players connected with an Ethereum wallet. If not speccified, it attaches the entity to the local player's avatar.
 - `anchorPointId`: What anchor point on the avatar skeleton to attach the entity, using a value from the enum `AvatarAnchorPointType`.
 
-The following anchor points are available on the player:
+The following anchor points are available on the `AvatarAnchorPointType` enum:
 
-- `AvatarAnchorPointType.AAPT_NAME_TAG`: Floats right above the player's name tag, isn't affected by the player's animations.
+- `AAPT_RIGHT_HAND`: Fixed on the player's right hand
+- `AAPT_LEFT_HAND`: Fixed on the player's left hand
+- `.AAPT_NAME_TAG`: Floats right above the player's name tag, isn't affected by the player's animations.
 
   > Note: The name tag height is dynamically adjusted based on the height of the wearables a player has on. So a player wearing a tall hat will have their name tag a little bit higher than others.
 
-- `AvatarAnchorPointType.AAPT_POSITION`: The player's overall position.
-- `AvatarAnchorPointType.AAPT_RIGHT_HAND`: Is fixed on the player's right hand
-- `AvatarAnchorPointType.AAPT_LEFT_HAND`: Is fixed on the player's left hand
+- `AAPT_POSITION`: The player's overall position.
+- `AAPT_HEAD`: Fixed to center of the player's head.
+- `AAPT_NECK`: Fixed to the player's base of the neck.
+- `AAPT_SPINE`: Fixed to the top section of the backbone.
```

Comment:
Maybe we can add that creators might want to add an offset when using this bone for positioning objects on the back or on the chest? And later on we could add a "Suggested offsets" table with some values for these and others?

Reply:
this is clarified and there's a snippet showing how to do this just a few lines below

A list of suggested offsets would be great, we need to come up with these values later